### PR TITLE
lightning: turn off shuffling for validation dataset

### PR DESCRIPTION
**Pull request by chongxiaoc**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/horovod/horovod/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs?
- [ ] Did you write any tests to validate this change?  
- [ ] Did you update the [CHANGELOG](https://github.com/horovod/horovod/blob/master/CHANGELOG.md), if this change affects users?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one member of the [technical steering committee](https://github.com/horovod/horovod/blob/master/CONTRIBUTING.md) must review and approve.
3. If any member of the technical steering committee requests changes, they must be addressed.

### CODE REVIEW

File: horovod/spark/lightning/remote.py [hunk 1]
```diff
@@ -222,7 +222,8 @@ def on_sanity_check_end(self, trainer, model):
 def _set_data_loader_fn(transformation, schema_fields, batch_size, calculate_shuffle_buffer_size, data_loader_cls, num_epochs, store, verbose=False):
 
     @contextlib.contextmanager
-    def set_data_loader(model, data_path, dataloader_attr, reader_worker_count, reader_pool_type, should_read=True, name="", limit_step_per_epoch=-1):
+    def set_data_loader(model, data_path, dataloader_attr, reader_worker_count, reader_pool_type,
+                        should_read=True, name="", limit_step_per_epoch=-1, is_val=False):
```
[the remark this answers is not in the document]
Comment (chongxiaoc) [reply horovod/spark/lightning/remote.py:226]: yep, sounds good

[the remark this answers is not in the document]
Comment (chongxiaoc) [reply horovod/spark/lightning/remote.py]: resolved.

**Review by irasit — APPROVED** [review]
